Source/PeleC.cpp

```diff
@@ -337,9 +337,14 @@ PeleC::read_params()
   }
 
   if (do_hydro) {
-    if (do_mol || ppm_type == 0) {
-      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4))
-        amrex::Error("PeleC::plm_iorder must be 1, 2, or 4");
+    if (ppm_type == 0) {
+      if ((plm_iorder != 1) && (plm_iorder != 2) && (plm_iorder != 4)) {
+        amrex::Error("PeleC::plm_iorder for ppm_type = 0 must be 1, 2, or 4");
+      }
+    } else if (do_mol) {
```

Comment:
Shouldn't `do_mol` be the `if` and `ppm_type` be the `else if` here?

Reply:
nice. good point. I think I fixed it?